src/Markup/Avalonia.Markup.Xaml.Loader/CompilerExtensions/AvaloniaXamlIlCompiler.cs

```diff
@@ -92,6 +92,12 @@ void InsertBefore<T>(params IXamlAstTransformer[] t)
                 new XamlMergeResourceGroupTransformer(),
                 new AvaloniaXamlIncludeTransformer()
             };
+
+
+            InsertBefore<AvaloniaXamlIlControlTemplatePriorityTransformer>(
+                new DependsOnTransformer()
+                );
```

Comment:
Can we please follow the convention for this? The entire project and all .NET doesn't do this. This is a new idea to simplify some git change tracking that is completely unnecessary. We should follow what .NET does.

You try to change this in almost every one of your PRs. We need to set a standard and all follow it. In the case of Avalonia and even .NET the standard here seems pretty clear.

Reply:
This is a preliminary PR, I'll reformat it after I'm done.